common/src/main/java/tachyon/worker/WorkerClient.java

```diff
@@ -78,18 +78,21 @@
   private final ClientMetrics mClientMetrics;
 
   /**
-   * Create a WorkerClient, with a given MasterClient.
+   * Creates a WorkerClient, with a given MasterClientBase.
    *
-   * @param masterClient a master client to retrieve user id and worker hostname info
-   * @param executorService an executor for the worker client's heartbeat thread
-   * @param conf the Tachyon Configuration to use
-   * @param clientMetrics metrics collection object for this client's operations
+   * @param workerNetAddress
+   * @param executorService
+   * @param conf
+   * @param userId
```

Comment:
I think we'd better do a global search, but perhaps not in this PR: it's used at many places, including files under client/